samples/msal-node-samples/refresh-token/adalCustomCache.js

```diff
@@ -0,0 +1,114 @@
+var _ = require('underscore');
+var fs = require('fs');
+
+/**
+ * Constructs a new token cache on disk.
+ * This effectively implements the ADAL TokenCache interface.
+ * @constructor
+ */
+function DiskCache(cacheLocation) {
+    this._cacheLocation = cacheLocation;
+    this._entries = [];
+}
+
+/**
+ * Removes a collection of entries from the cache in a single batch operation.
+ * @param {Array} entries An array of cache entries to remove.
+ * @param {Function} callback This function is called when the operation is complete. Any error is provided as the first parameter.
+ */
+DiskCache.prototype.remove = function (entries, callback) {
+    const cachedEntries = _readCache.call(this);
+
+    var updatedEntries = _.filter(cachedEntries, function (element) {
```

Comment:
I had previously commented const vs var. Why aren't you using var on line 20?

Reply:
ah, my bad, this one slipped through.. updated!